src/sass/chrome.scss

```diff
@@ -69,6 +70,33 @@ aside {
     }
 }
 
+// Beta Banner
+.ins-c-page__beta {
+    background: $bg-betabar;
+    width: 100vw;
+    height: 50px;
+    justify-content: center;
+    grid-area: beta;
+
+    .pf-c-button {
+        margin: auto 1em;
+        //background-color: var(--pf-c-button--m-primary--BackgroundColor) !important;
+        //color: var(--pf-c-button--m-primary--Color) !important;
+    }
+
+    i {
+        padding-right: 6px;
+    }
+}
+
+.ins-c-page_beta-hidden {
+    display: none !important;
```

Comment:
Try not to ever use `!important`. Also, this should be overwritten by the conditional rendering